fuel-derive/manual_impl_tx_serde.rs

```diff
@@ -0,0 +1,77 @@
+impl fuel_types::canonical::SerializedSize for Transaction {
```

Comment:
How does this file get used? I can't find it being imported anywhere with a quick `rg` search over the files.

Reply:
if it's just for local testing, maybe add a comment explaining what it's for

Reply:
It's not needed anymore. Removed.